stock_pack_operation_auto_fill/models/stock_move.py

```diff
@@ -8,12 +8,25 @@
 class StockMove(models.Model):
     _inherit = 'stock.move'
 
+    def _create_extra_move(self):
+        """
+        When user set on stock move line done a quantity greater than initial
+        demand Odoo creates an extra stock move with the difference and it is
+        posible to create an extra stock move line with qty_done = 0 which will
+        be deleted in _action_done method.
+        This method set a context variable to prevent set qty_done for these
+        cases.
+        """
+        my_self = self.with_context(skip_auto_fill_operation=True)
+        return super(StockMove, my_self)._create_extra_move()
+
     def _prepare_move_line_vals(self, quantity=None, reserved_quant=None):
         """Auto-assign as done the quantity proposed for the lots"""
-        self.ensure_one()
```

Comment:
No need to remove this

Reply:
Yes, I know, but the method already has this check.
https://github.com/odoo/odoo/blob/963475bcfd75fe0a35401c7343d14937aa7f0d74/addons/stock/models/stock_move.py#L810

Reply:
The call is not for doing the check, but for raising the error on a meaningful line, at the beginning on the method, instead of a weird line where it's more difficult to know which one should be a singleton. Removing this line here, the error will raise in the super, and thus maybe you are not aware that the problem is in the previous call. Just this case it's not critical, but for you to know why the call.